src/main/java/core/ServerRequestHandler.java

```diff
@@ -18,11 +18,28 @@
  */
 @Slf4j
 public class ServerRequestHandler {
-    private static final int SERVER_PORT = 7080;
+    private static int SERVER_PORT = 7080;
```

Comment:
if you remove final, so this variable can change your value. Should you remove "static" clause too?